tasks/insights-client.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: MIT
+---
```

Comment:
The code in this file looks very similar to tasks/insights-client-unregistration.yml - can you refactor into a single file?

Reply:
Not too much -- I think the biggest duplication here is the query `insights-client --status`. I think we can move that to an helper yml file that registers an internal variable, so the logic is done in only one place. This is similar to what I did in some of the tests, see some of the `tests/tasks/*.yml` files.